dotnet-logging/LogLevelsWithSerilog/LogLevelsWithSerilog/LogLevelsWithSerilog.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
```

Comment:
Should be targetting .NET 7.0. We want to target the most recent released/supported framework.

Reply:
I updated my code